tools/experiments/experiments.js

```diff
@@ -329,7 +329,7 @@ const EXPERIMENTS = [
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/17243',
   },
   {
-    id: 'amp-carousel-scroll-snap',
+    id: 'amp-carousel-chrome-scroll-snap',
```

Comment:
s/`Enables scroll snap on carousel across all browsers/OSes`/ `Enables scroll snap on carousel on Chrome browsers`

Reply:
I guess a separate PR to remove the old name from Canary config.